livekit-agents/livekit/agents/llm/llm.py

```diff
@@ -58,7 +59,7 @@ class ChatChunk(BaseModel):
 
 class LLM(
     ABC,
-    rtc.EventEmitter[Union[Literal["metrics_collected"], TEvent]],
+    rtc.EventEmitter[Union[Literal["metrics_collected"], Literal["error"], TEvent]],
```

Comment:
I think this should work
```suggestion
    rtc.EventEmitter[Union[Literal["metrics_collected", "error"], TEvent]],
```